week-3/F-array-forEach/exercise.js

```diff
@@ -8,7 +8,17 @@
 */
 
 var arr = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
-
+arr.forEach(num => {
+  if (num % 5 === 0 && num % 3 === 0 && num !== 0) {
+    console.log("FizzBuzz");
+  } else if (num % 5 === 0 && num !== 0) {
+    console.log("Buzz");
+  } else if (num % 3 === 0 && num !== 0) {
+    console.log("Fizz");
+  } else {
+    console.log(num);
+  }
+});
```

Comment:
Good! Well done for checking the `0` as well! 
A more efficient way of doing this though would be to nest the if statements - you can start with checking if it's a `0`, and if it's not - proceed with all the other checks. Try that :)